ptvsd/debugger.py

```diff
@@ -14,7 +14,7 @@
 
 def debug(filename, port_num, debug_id, debug_options, run_as, **kwargs):
     # TODO: docstring
-    address = (None, port_num)
+    address = ('127.0.0.1', port_num)
```

Comment:
Any particular reason to not use `'localhost'` here? That should (theoretically) be better in the presence of IPv6 networks.

Reply:
👍